SignalServiceKit/src/Protos/OWSSignalServiceProtos.pb.swift

```diff
@@ -0,0 +1,4199 @@
+// DO NOT EDIT.
```

Comment:
I see that you included OWSSignalServiceProtos.pb.swift.  What about the other 3 generated swift files?

Reply:
I verified that they generated. I could include them, but they aren't being used yet.